docs/configuration/git-providers/gitlab-cloud.md

```diff
@@ -1 +1,33 @@
 # GitLab Cloud
+
+## GitLab Application
+
+Follow the instructions below to set up the Codacy integration with GitLab Cloud:
+
+1.  Create an application pointing to your local Codacy deployment URL.
```

Comment:
Can we provide info on how to create the GitLab Application? If we can provide the direct URL to the Applications screen on GitLab it should be enough:

```suggestion
1.  Create a [new application](https://gitlab.com/oauth/applications) on GitLab pointing to your local Codacy deployment URL.
```

Reply:
We should have the same experience we had in the previous docs. I placed the link in a previous comment

Reply:
@rtfpessoa is right, we're also missing the scopes that must be checked, for instance. :+1: